config/webpack.tests.config.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const glob = require('glob');
+const path = require('path');
+const nodeExternals = require('webpack-node-externals');
+const CopyPlugin = require('copy-webpack-plugin');
+
+const OUT_TEST_DIR = path.resolve(__dirname, '..', 'out', 'test');
+const TEST_DIR = path.resolve(__dirname, '..', 'test');
```

Comment:
Instead of outputting to `root/dist`, this config outputs testing related things to `root/out/test`. I didn't switch the directory of the main extension entry point from `root/dist`, but lmk if you think that would be a good idea and I can look into that